README.md

```diff
@@ -184,3 +184,29 @@ For OSP deploy with Ceph using Composable Roles, After setting the above specifi
 
 Note: User can customize [internal.yml.j2](templates/internal.yml.j2) template for Ceph deployment based on their
       requirement if needed
+2) set composable_roles: true true in group_vars/all.yml
+
+## Pbench agent Installations
+To install Pbench agent on the compute nodes and vm's (i.e client and server),you have to follow the below steps and set the below params install/group_vars/all.yml
+
+`pbench_key_url: <pbench_key_url>
+ pbench_config_url: <pbench_config_url>
+ pbench_configuration_environment: <pbench_configuration_environment>`
+
+`cd  {{ ansible_user_dir}}/jetpack/install`
+
+1) To create the client and server
+     `ansible-playbook boot.yml`
+
+To install Pbench agent on Trafficgen host, set the below params
+
+trafficgen_host:
+  ip: <ip>
+  ssh_pass: password
+  ssh_user: root
+
+2) Install Pbench agent bootstrapping
+     `ansible-playbook pbench_agent_bootstrap.yml -e "repos=<perf-dept-repo>" `
```

Comment:
As mentioned below, this feels more like a generic bootstrap step than anything specific for pbench agent installation.